packages/x-date-pickers/src/locales/zhHK.ts

```diff
@@ -65,7 +65,7 @@ const zhHKPickers: Partial<PickersLocaleText<any>> = {
     value !== null && utils.isValid(value)
       ? `選擇時間，已選擇${utils.format(value, 'fullTime')}`
       : '選擇時間',
-  // fieldClearLabel: 'Clear value',
+  fieldClearLabel: '清除',
```

Comment:
The suggested translation seems to mean "Clear" in English.
Are you sure it is enough in this case or would it be better to use `明确的价值` instead? 🤔 
```suggestion
  fieldClearLabel: '明确的价值',
```

Reply:
I guess it's OK, it's what simplified Chinese uses 

https://github.com/mui/mui-x/blob/b4e8bf4bef9349a3444b64fcea1c0b7c77060ee9/packages/x-date-pickers/src/locales/zhCN.ts#L68

---

Off-topic: this is strange

<img width="642" alt="SCR-20240817-brju" src="https://github.com/user-attachments/assets/439f56a8-f37d-4ac5-a142-706e3d3cfde2">

https://mui.com/material-ui/react-autocomplete/#combo-box

<img width="636" alt="SCR-20240817-brla" src="https://github.com/user-attachments/assets/937ea812-7edf-49e0-88f2-a5e5ff191306">

https://mui.com/x/react-date-pickers/date-picker/#clearing-the-value

Reply:
Thank you for the insight. 👍 
This is the discussion regarding this decision: https://github.com/mui/mui-x/pull/9095#discussion_r1337063290
If we go with only `Clear`, I'd say that we need to explore adding an `aria` relationship with the "Field" label to have something like `Clear Birth date, button` on screen readers. 💡 🤔

Reply:
> This is the discussion regarding this decision: https://github.com/mui/mui-x/pull/9095#discussion_r1337063290

Interesting, but wait, why is the clear button reachable with <kbd>Tab</kbd>? I would expect the same as in the autocomplete: https://github.com/mui/material-ui/issues/30815

https://react-spectrum.adobe.com/react-aria/ComboBox.html#state uses "Clear" only. https://react-select.com/home makes it unreachable.

Reply:
> Interesting, but why is the clear button reachable with Tab? I would expect the same as in the autocomplete: [mui/material-ui#30815](https://github.com/mui/material-ui/issues/30815)

Why would we hide it? 🤔 
It is a functional part of the component, I don't see a reason to forbid its usage on screen readers (or with a keyboard). 🙈 


> https://react-spectrum.adobe.com/react-aria/ComboBox.html#state uses "Clear" only. https://react-select.com/home makes it unreachable.

So does the [clearable date picker](https://react-spectrum.adobe.com/react-aria/DatePicker.html#state). 👍 
However, they don't even provide a `title`. In this regard I do not agree, I feel that `title` is a superior attribute to `aria-label`, which should be used when the title doesn't make sense.

Reply:
> Why would we hide it?

WAI-ARIA recommends it: https://github.com/mui/material-ui/issues/30815#issuecomment-1028935533 and in https://github.com/mui/material-ui/issues/30815#issuecomment-2296573576 I tried to justify the rational behind why I built the Autocomplete this way.

Reply:
I feel like we are comparing different things—autocomplete/select has a different keyboard behavior to the Pickers/Field.
It is way easier to remove a value with a keyboard from the autocomplete as compared to a Pickers Field. 🤔

Reply:
I don't know, on a keyboard clearing a date picker feels slightly simpler than on autocomplete, either:
 - I tab into it. In both cases, everything is selected, so pressing delete empty.
 - I have just selected a value. In both cases, I select all + press delete
 - I have just selected a value. I spam the delete key, the autocomplete usually has more characters to remove a bit slower.

Reply:
- Tab discussion moved to https://github.com/mui/mui-x/issues/14587
- Label PR merged https://github.com/mui/mui-x/pull/14243